source/the-commodities-api.html.md

```diff
@@ -378,6 +378,253 @@ Look for the unique identifier on an entity of type `additional_code` in the `in
 }
 ```
```

Comment:
How comes this is all red in the syntax highlight? Partial display trying to be rendered in github?